packages/block-library/src/video/edit.js

```diff
@@ -232,11 +240,21 @@ class VideoEdit extends Component {
 											isDefault
 											onClick={ open }
 											ref={ this.posterImageButton }
+											aria-describedby={ `video-block__poster-image-description-${ instanceId }` }
 										>
 											{ ! this.props.attributes.poster ? __( 'Select Poster Image' ) : __( 'Replace image' ) }
 										</Button>
 									) }
 								/>
+								<p
+									id={ `video-block__poster-image-description-${ instanceId }` }
+									hidden
```

Comment:
What's the purpose of using `hidden` prop here? In other places, we use [screen-reader-text](https://make.wordpress.org/accessibility/handbook/markup/the-css-class-screen-reader-text/) class name if the message exists only for screen readers.

Reply:
Hi @gziolo, nice question!
Like you said we use screen-reader-text class name if the message exists only for screen readers. When we use screen-reader-text the screen readers still announce the text of the element. In this case, we also want to hide this element from screen readers. This element should not appear on the a11y tree by itself. It is only used when the other element that references it is focused and in that case the description is announced as part of the other element because it references it with aria-describedby.

Reply:
I didn’t articulate it good enough. This special class hides an element completely unless it receives focus. In this case this class would be applied to p element which never gets focused through tabbing.

Reply:
@joedolson, could you advise what is the best approach? I saw your name in related docs 😃

Reply:
Just trying to figure out exactly what this change is doing. It seems like it's adding a description of the image that's currently selected as the poster image - is that correct? 

If so, this should be fine. If a hidden element is the target of an aria-describedby or aria-labelledby attribute, that text should be read out by screen readers when they focus on the element that carries the labelling attribute. (In short: being hidden doesn't matter to aria-describedby).

Do I correctly understand what's being done here?

Reply:
Yes, it’s correct. Thanks for a quick answer 👍